backend/nginx/nginx.conf

```diff
@@ -0,0 +1,13 @@
+upstream linked_commons {
+    server backend:8000;
+}
+
+server {
+    listen 80;
+    location / {
+        proxy_pass http://linked_commons;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $host;
+        proxy_redirect off;
+    }
+}
```

Comment:
Is this still needed?

Reply:
No. It's no longer needed. Just removed them.

Reply:
No. There are no longer needed. Just removed them.

Reply:
No. These are no longer needed. Just removed them.